app/viewmodels/RepeaterAnswerSection.scala

```diff
@@ -20,4 +20,6 @@ case class RepeaterAnswerSection (headingKey: String,
                                   relevanceRow: AnswerRow,
                                   rows: Seq[RepeaterAnswerRow],
                                   addLinkKey: String,
-                                  addLinkUrl: String) extends Section
+                                  addLinkUrl: String) extends Section {
+  override def isEmpty: Boolean = rows.isEmpty
```

Comment:
is the file name (and the class name) correct?
`RepeaterAnswerSection`?
Did you mean `Repeated`?

Reply:
I don't know.  It's an existing class.  Don't think we use it.

Reply:
You could fix this typo

Reply:
we could check what is the purpose of these classes and do some cleaning